user-stories.md

```diff
@@ -0,0 +1,8 @@
+# Пользовательские истории:
+- Как пользователь, я хочу зарегистрироваться и получить свой аккаунт с уникальным id, чтобы меня можно было отличить от других пользователей.
+- Как пользователь, я хочу получать короткую уникальную ссылку по длинному URL, чтобы было проще её использовать.
+- Как пользователь, я хочу использовать выданную мне короткую ссылку и переходить на соответствующий сайт без задержек.
+- Как зарегистрированный пользователь, я хочу иметь возможность просматривать уже созданные мною ранее сокращения для ссылок, чтобы использовать их повторно.
+- Как зарегистрированный пользователь, я хочу иметь возможность удалять созданные мною ранее сокращения для ссылок.
+- Как зарегистрированный пользователь, я хочу, чтобы действия от моего аккаунта мог выполнять исключительно я, чтобы никто не смог изменить мои ссылки.
+- Как зарегистрированных пользователь, я хочу чтобы созданные мною ранее ссылки и их полные версии видел исключительно я, чтобы обеспечить приватность. 
```

Comment:
If I place my short link in some social media, how does your service will protect my short link?
Do you mean that only I as a user could follow my short links?